### PR TITLE
New Entry: matplotlib.pyplot.pie()

### LINKED ISSUE #1661: [Term Entry] Matplotlib `pyplot`: `pie`
### Reason/inspiration (optional)

Please write a new entry under:  content/matplotlib/concepts/pyplot/terms/ about the [`.pie()`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.pie.html) method in Python.

A "Syntax" section with pseudo code snippet.
An "Example" section with a Python code snippet. If images are included, they should go in the media/ folder.

To get started with the PR and entry file:

Under content/matplotlib/concepts/pyplot/terms/, create a new folder called `pie`. Then create a new file called `pie.md` under content/matplotlib/concepts/pyplot/terms/`pie`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Nate1809**
### Description

Closes #1661.

Created entry for the **`.pie()`** method in the matplotlib library.
I added the pie.md file in content/matplotlib/concepts/pyplot/terms/pie and added 2 example pictures in the /media/ directory.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+A simple pie chart: 
+
+```py
+
+# Import libraries
+from matplotlib import pyplot as plt
+
+# Creating dataset
+pizza = ['CHEESE','PEPPERONI','VEGGIE','MEAT','BBQ CHICKEN','HAWAIIAN']
+data = [23,  17,  35,  29,  12,  41]
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:78]: these values add up to greater than 100

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+Customizing the pie chart:
+
+```py
+
+# Import libraries
+import matplotlib.pyplot as plt
+
+# Plot data
+pizza = ['CHEESE','PEPPERONI','VEGGIE','MEAT','BBQ CHICKEN','HAWAIIAN']
+popularity = [22.2,  17.6,  8.8,  8,  7.7,  6.7]
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:102]: these values don't add up to 100

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Output:
+
+![Output of matplotlib.pyplot.pie() function example 1](/media/matplotlib-pie-1.png)
+
+
+Customizing the pie chart:
+
+```py
+
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:96]:
```suggestion
```

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  ## Examples
+
+ 
+Examples below demonstrate the use of `.pie()` to plot values and show the functionality of some arguments.
+
+A simple pie chart: 
+
+```py
+
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:72]:
```suggestion
```

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+# Creating dataset
+pizza = ['CHEESE','PEPPERONI','VEGGIE','MEAT','BBQ CHICKEN','HAWAIIAN']
+data = [23,  17,  35,  29,  12,  41]
+
+# Creating plot
+plt.pie(data, labels = pizza)
+
+# show plot
+plt.show()
+
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:85]:
```suggestion
```

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+#colors = ['red', 'green', 'yellow', 'blue', 'purple', 'brown']
+colors = ['#FF6188',  '#A9DC76',  '#FFD866',  '#78DCE8',  '#AB9DF2',  '#8c564b']
+
+# explode 1st slice
+explode = (0.1,  0,  0,  0,  0,  0)
+
+# Plot
+plt.pie(popularity, explode=explode, labels=pizza, colors=colors, autopct='%1.1f%%', shadow=True, startangle=140)
+plt.show()
+
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:113]:
```suggestion
```

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+```
+
+Output:
+
+![Output of matplotlib.pyplot.pie() function example 2](/media/matplotlib-pie-2.png)
+
```
Comment (caupolicandiaz) [inline content/matplotlib/concepts/pyplot/terms/pie/pie.md:119]: please update images after making the changes above and delete images no longer in use.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Nate1809 I've finished my initial review. Please update and tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@Nate1809 There hasn't been any movement on this entry, do you need any help with this?

**Comment by SSwiniarski** [comment]
@caupolicandiaz- I don't think the data needs to add up to 100, it wouldn't in the wild. The pie just shows the proportions with respect to the total.

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md (hunk 1 above)
Comment (SSwiniarski) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:59]: They shouldn't have to.

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md (hunk 2 above)
Comment (SSwiniarski) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:81]: Again, they shouldn't have to.

File: content/matplotlib/concepts/pyplot/terms/pie/pie.md (hunk 1 above)
Comment (caupolicandiaz) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:59]: Respectfully, I must insist this is incorrect. If you have a poll in which you ask everyone for a single "favorite" topping you will always get values that add up to 100 (e.g. 40% pepperoni , 30% sausage, 25% mushroom, 5% cheese). If you have data that does not add up to 100, it should not be displayed with a pie plot. Using a pie plot in circumstances in which the data does not add up to 100 is a visualization no-no.

Comment (SSwiniarski) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:59]: What if you're counting the number of pizzas sold and you want to have the pie chart to display the percentage of each? You're asking the developer to do the math that `.pie()` does automatically for you.

Comment (caupolicandiaz) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:59]: Technically, this first example is okay. However, the second example uses the following data:
popularity = [22.2,  17.6,  8.8,  8,  7.7,  6.7]
The name of the variable and the floating point values suggest percentages. But, these values do not add up to 100. If you have vote counts representing the popularity of toppings there should only be whole numbers. It's impossible to have .2 of a vote. Therefore, the data doesn't make sense in this context.

Comment (SSwiniarski) [reply content/matplotlib/concepts/pyplot/terms/pie/pie.md:59]: I've updated the second example to use whole numbers.

**Comment by SSwiniarski** [comment]
@caupolicandiaz I updated the second example to use whole numbers. We need a new image for the second pie chart, but I can't upload to this branch, so I'll have to add it in a second PR.

**Review by caupolicandiaz — APPROVED** [review]
@SSwiniarski @Nate1809 I've updated the plots, LGTM

**Review by SSwiniarski — APPROVED** [review]
@caupolicandiaz, @Nate1809 Looks good to me.

**Comment by SSwiniarski** [comment]
@Nate1809 congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/matplotlib/pyplot/pie

**Pull request merged**